tests/Integration/PrestaShopBundle/Routing/Converter/LegacyUrlConverterTest.php

```diff
@@ -144,7 +144,7 @@ public static function getMigratedControllers(): array
             'admin_geolocation_by_ip_address_save' => ['/improve/international/geolocation/by-ip-address', 'AdminGeolocation', 'update'],
 
             'admin_payment_preferences' => ['/improve/payment/preferences', 'AdminPaymentPreferences'],
-            'admin_payment_preferences_process' => ['/improve/payment/preferences', 'AdminPaymentPreferences', 'update'],
+            'admin_payment_carrier_restrictions_save' => ['/improve/payment/preferences', 'AdminPaymentPreferences', 'update'],
 
```

Comment:
I am not sure if's needed or should I remove it completely. Technically the old logic from legacy for saving payment preferences is gone.

Reply:
I guess its ok, it can be removed when the legacy controller is removed.

Reply:
It actually already has been removed during the first migration of this page, actually the purpose of this test was to ensure old controllers were correctly assigned with their new symfony routes, but we never kept it up to date so it kind of lost its purpose over time

Anyway don't worry too much about this one, as long as there is one route matching the index of the page it's ok you could remove this extra line if you want